packages/contracts-bedrock/src/universal/CrossDomainMessenger.sol

```diff
@@ -358,6 +362,15 @@ abstract contract CrossDomainMessenger is
         + RELAY_GAS_CHECK_BUFFER;
     }
 
+    /// @notice Returns the address of the custom gas token and the token's decimals.
```

Comment:
nit, since gas token can be non-custom also (ETH)
```suggestion
    /// @notice Returns the address of the gas token and the token's decimals.
```

Reply:
This has been addressed